apps/authorization/models.py

```diff
@@ -70,11 +73,217 @@ def update_grants(*args, **kwargs):
 
 def check_grants():
     AccessToken = get_access_token_model()
-    token_count = AccessToken.objects.filter(
-        expires__gt=timezone.now(),
-    ).values('user', 'application').distinct().count()
+    token_count = (
+        AccessToken.objects.filter(
+            expires__gt=timezone.now(),
+        )
+        .values("user", "application")
+        .distinct()
+        .count()
+    )
     grant_count = DataAccessGrant.objects.all().count()
     return {
         "unique_tokens": token_count,
         "grants": grant_count,
     }
+
+
```

Comment:
START-CODE-FOR-REVIEW-SECTION

Creates `get_grant_counts()` and `get_grant_by_app_counts(application)` functions that returns `DataAccessGrant` and `ArchivedDataAccessGrant` related model counts.